easy-ui-tokens/package.json

```diff
@@ -11,10 +11,17 @@
     "./scss/*": "./dist/scss/*",
     "./json/*": "./dist/json/*"
   },
+  "publishConfig": {
+    "access": "public",
+    "directory": "dist"
+  },
   "scripts": {
-    "build": "style-dictionary build"
+    "build": "run-p build:*",
+    "build:tokens": "style-dictionary build",
+    "build:pkg": "node ../scripts/copyPkgJsonForDist.mjs",
+    "clean": "rm -rf .turbo dist node_modules"
   },
-  "dependencies": {
+  "devDependencies": {
```

Comment:
Double checking on whether this was an intentional change

Reply:
it was lol. thanks for checking